plugins/template.md

```diff
@@ -356,6 +356,12 @@ This is a simple <strong>snippet</strong>. Will be replaced: {$somevar1}.
 
 {% include misc/plugin-menu-item-id-boilerplate.md %}
 
+## Example Configuration
```

Comment:
Move this section to immediately above the Configuration Options heading and change the heading to "Live example"

Reply:
Not title case? "Live Example"?